src/scripts/choices.test.js

```diff
@@ -15,6 +15,22 @@ describe('choices', () => {
     });
   };
 
+  describe('constructor can return array of instances', () => {
+    it('should be the same as number of matched elements', () => {
+      document.body.innerHTML = `
+      <input data-choice type="text" id="inp1" />
+      <input data-choice type="text" id="inp2" />
+      <input data-choice type="text" id="inp3" />
+      `;
+
+      const inputs = document.querySelectorAll('[data-choice]');
+      expect(inputs.length).to.equal(3);
+
+      const choices = new Choices();
```

Comment:
```suggestion
      const choices = new Choices(inputs);
``` 
:)

Reply:
The whole point here is to construct from a string. If a pass nothing to Choices constructor then defaults is a string ‘[data-choice]’

Reply:
By the way, `new Choices(inputs)` will not work - Choices is not supporting directly `NodesList` as constructor parameter.

Reply:
The thing is that if the constructor never worked in the expected way with selector returning several elements, then you can adjust/fix its behaviour in a minor version as it cannot be considered as breaking change.